client/.env.example

```
@@ -3,6 +3,6 @@
 VITE_GOOGLE_CLIENT_ID=
 VITE_DODO_MODE=test_mode
 # API endpoint (for local dev outside Docker)
-VITE_API_URL=http://localhost:3000
+VITE_API_URL=http://localhost:3000/api
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_API_URL key should go before the VITE_DODO_MODE key

(UnorderedKey)